model-api-gen-gradle-test/build.gradle.kts

```diff
@@ -108,6 +110,19 @@ metamodel {
     registrationHelperName = "org.modelix.apigen.test.ApigenTestLanguages"
 }
 
+@OptIn(UnstableGeneratorFeature::class)
+tasks.register<UseExportedLanguagesTask>("generateUsedLanguagesSummary") {
+    val outputFile = buildDir.resolve("languagesSummary.txt")
+    outputs.file(outputFile)
+    dependsOn("exportMetaModelFromMps")
+    includedLanguages.set(listOf("jetbrains.mps.lang.core"))
+    doWithLanguages { languages ->
+        // An actual user case would extract the custom generator into its own class.
```

Comment:
The main use case of this feature is to use the generated languages for custom generators.

Currently, prototyping generators needs directly modifying the plugin.
And after prototyping the generator, it must become part of the offical plugin or a fork has to be maintained.
See. https://github.com/abstraktor/modelix.core/tree/feature/VuejsMMGenerator-proxy

I think this feature is a good escape hatch. Because I'm not certain about the API, and it exposes internal structures almost directly, I marked it as `UnstableGeneratorFeature`.

For example, I will use this escape hatch to develop (at least the first iteration of) samples with Vue and React.

I'm not certain such an escape hatch is maybe feature creep. Perhaps users should rather the generation capacities of MPS. I am curious about your opinion.